Extensions/xkit_patches.js

```diff
@@ -1,7 +1,7 @@
 //* TITLE XKit Patches **//
-//* VERSION 5.0.0 **//
+//* VERSION 5.0.1 **//
```

Comment:
Per semver, since functionality is being added, should be at least a `MINOR` patch version increase.